SuperBuild/External_tinyxml2.cmake

```diff
@@ -37,6 +37,7 @@ if(NOT DEFINED ${proj}_DIR AND NOT ${SUPERBUILD_TOPLEVEL_PROJECT}_USE_SYSTEM_${p
       -DCMAKE_CXX_STANDARD:STRING=${CMAKE_CXX_STANDARD}
       -DCMAKE_CXX_STANDARD_REQUIRED:BOOL=${CMAKE_CXX_STANDARD_REQUIRED}
       -DCMAKE_CXX_EXTENSIONS:BOOL=${CMAKE_CXX_EXTENSIONS}
+      -DCMAKE_OSX_ARCHITECTURES:STRING=${CMAKE_OSX_ARCHITECTURES}
```

Comment:
Adding this should not be required, instead consider adding `${${proj}_EP_ARGS}` to all external project like it is done [here](https://github.com/Slicer/Slicer/blob/b2edf47475d01322d4ce96847a4d162289ef3c8a/Utilities/Templates/Extensions/SuperBuild/SuperBuild/External_Foo.cmake#L38-L39).